glider_kinematics_gazebo_plugins/kinematics_plugins/include/kinematics_plugins/GaussMarkovProcess.hh

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2016 The UUV Simulator Authors.
```

Comment:
Is this copied from the UUV Simulator?   Can we add the appropriate UUV simulator package as a dependency and avoid copying the source?